client/gutenberg/extensions/slideshow/view.js

```diff
@@ -0,0 +1,24 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+
+import './style.scss';
+import component from './component.js';
+import { settings } from './settings.js';
+import FrontendManagement from 'gutenberg/extensions/shared/frontend-management.js';
+
+typeof window !== 'undefined' &&
+	window &&
```

Comment:
I am not sure but doing isn't doing 
```
typeof window !== 'undefined' &&
``` and 
```
	window &&
``` 
a bit redundant?

Reply:
@sirreal originally suggested this change, so I'd be curious to hear from him if there is also a scenario where window object exists but is null? I'll assume not and remove the duplication, but flagging here just in case there is such a situation that should be checked. Removed in https://github.com/Automattic/wp-calypso/pull/29419/commits/aa0842c85f826929fbc46f144c600504a9a4021e